manga_py/providers/__init__.py

```diff
@@ -453,6 +453,9 @@
     'manganelo_com': [
         r'manganelo\.com/(manga|chapter)/.',
     ],
+    'manganato_com': [
+        r'readmanganato\.com/manga-.',
```

Comment:
I had to add another line to get some of the urls to work.

```
    'manganato_com': [
        r'manganato\.com/manga-.',
        r'readmanganato\.com/manga-.',
    ],
```

Your PR worked great otherwise!